pages/dataset/[datasetid].js

```diff
@@ -63,286 +77,309 @@ const Dataset = ({ dataset }) => {
             </div>
           </div>
         </div>
-        {dataset.description != '' &&
-        (
-          <div>
-            <div className="flex mb-10">
-              <h2 className="mr-10 font-lato font-bold text-xl">
-            About this dataset
-              </h2>
-            </div>
-            <div className="mb-20 font-karla text-lg">{dataset.description}</div>
-          </div>
-        )}
-        <h2 className="mb-10 font-lato font-bold text-xl">File preview</h2>
+        <div className="flex mb-10">
+          <h2 className="mr-10 font-lato font-bold text-xl">
+            About this Dataset
+          </h2>
+          {/* <a href="#">
+            <img src="/share.svg" alt="next" />
+          </a> */}
+        </div>
+        <div className="mb-20 font-karla text-lg">{dataset.description}</div>
+        <h2 className="mb-10 font-lato font-bold text-xl">File Preview</h2>
         <div className="mb-10">
-          {data && data.length ? (
+          {data && data.length != 0 ? (
             <CustomTable data={data} columns={columns} />
           ) : (
-            'No preview is available for this dataset.'
+            'NO PREVIEW FOR THIS DATASET'
```

Comment:
@risenW Can we keep the text `No preview is available for this dataset.` instead, please? :slightly_smiling_face:

Reply:
Okay, will update it